Docs/source/Hardware/battery-compartment.rst

```diff
@@ -0,0 +1,9 @@
+Battery compartment
+===================
+
+The battery compartment has two shelves of battery modules. Each shelf has two rows of modules for a total of four in the compartment.
```

Comment:
I don't think this is still accurate, since I don't think this has been updated since the most recent battery box design (January 2021). Please confirm with Mahesh Govindu (enclosures lead)